dvc/command/get.py

```diff
@@ -10,7 +10,28 @@
 
 
 class CmdGet(CmdBaseNoRepo):
+    def _run_get_url(self):
+        from dvc.api import get_url
+
+        try:
+            url = get_url(
+                self.args.path, repo=self.args.url, rev=self.args.rev
+            )
+            logger.info("{} {}".format(self.args.path, url))
+        except DvcException:
+            logger.exception(
+                "failed to get url for '{}' from '{}'".format(
```

Comment:
Maybe
```suggestion
                "failed to show url for '{}' from '{}'".format(
```
so it is more in-check with `--show-url`?